tests/integration/ssr.test.js

```diff
@@ -5,7 +5,7 @@ const cheerio = require("cheerio");
 const chai = require("chai");
```

Comment:
Does this test need to be in `mocha`? 
I believe this same test in jest would look pretty similar.
```js
test("SSR Loads with an HTML Body", () => {
  request(url, (err, resp, body) => {
      if (!err && resp.statusCode === 200) {
        const cheer = cheerio.load(body);
        expect(cheer("#__next").find("div")).toBeGreaterThan(0);
      }
    });
});
```
If we can I'd like to do these in Jest so we can remove the deps on `mocha`, `chai`, and `chai-cheerio`.

Reply:
I will try that for a next PR, have not used Jest before to scrape through HTML.